controller/finalizer_test.go

```diff
@@ -137,103 +137,6 @@ func Test_createAddFinalizerPatch(t *testing.T) {
 	}
 }
 
-func Test_createRemoveFinalizerPatch(t *testing.T) {
```

Comment:
The function got removed and now there is not a good way to test the patch creation.